KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -506,6 +506,12 @@ public enum UITexts {
         static let balanceTitle = "kh_uisdk_loyalty_balance_title".localized
         static let or = "kh_uisdk_loyalty_separator".localized
     }
+    
+    public enum Accessibility {
```

Comment:
I am not sure about this approach – if you would prefer to go for it, we need to move all accessibility-related keys/values to this enum.

Reply:
It would be nice to have all accessibility strings in one place to find them more easily and avoid duplicates like the one added for closing the screen (which I'll remove shortly). I also encourage the use of dedicated accessibility strings and not reuse any existing strings that were created for other purposes to avoid unintended issues in case of retranslation. There would be some value duplication, but at least any changes to the voice over texts would be intended and not accidental.

For now I suggest we keep this demarcation and I move the 3 acc keys here too (and cleanup the path wherever they are being used in code). This would not involve renaming any pre-existing keys in Phrase. 

Also, it would be nice to have a ticket to create dedicated accessibility keys for all the elements and add them here.

WDYT?

Reply:
Sure, I agree. Although, I'd replace `acc` with full `accessibility` in key text. I find shortages problematic in a long term.

Reply:
Done. Left the existing keys as they were to avoid issues with Android, they should be tackled in a dedicated ticket (already created). But I renamed the new keys to avoid the abbreviation.